build/packager.js

```diff
@@ -175,6 +184,11 @@ Packager.prototype.includePackagedModules = function (next) {
 		|| supportedPlatforms.indexOf('ipad') !== -1) {
 		supportedPlatforms = supportedPlatforms.concat([ 'ios', 'iphone', 'ipad' ]);
 	}
+
+	// Hyperloop has no single platform downloads yet, so we use a fake platform
+	// that will download the all-in-one distribution.
+	supportedPlatforms = supportedPlatforms.concat([ 'hyperloop' ]);
```

Comment:
so you can just hijack the "commonjs" entry to mean "All platforms" too. That's what I did locally when I took a look at this.